scroll.go

```diff
@@ -0,0 +1,18 @@
+package fyne
+
+// ScrollDirection represents the directions in which a scrollable container or widget can scroll its child content.
+type ScrollDirection int
+
+// Constants for valid values of ScrollDirection used in containers and widgets.
+const (
+	// ScrollBoth supports horizontal and vertical scrolling.
+	ScrollBoth ScrollDirection = iota
+	// ScrollHorizontalOnly specifies the scrolling should only happen left to right.
+	ScrollHorizontalOnly
+	// ScrollVerticalOnly specifies the scrolling should only happen top to bottom.
+	ScrollVerticalOnly
+	// ScrollNone turns off scrolling for this container.
+	//
+	// Since: 2.6
```

Comment:
All of these above are new within the `fyne` package scope. I understand why this was added here but it is slightly confusing 😅

Reply:
I get your point but what would be more correct? I can switch to whatever you suggest, but I'm not sure what is right.

Reply:
I'd suggest adding the line to the type only as it is new in this release but I don't know really.

Reply:
That works, done thanks :)